apps/docs/docs/user/runtime-parameters.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 4
+---
+
+# Runtime Parameters
+
+Property values in Jayvee can be assigned to `values` or left open for later configuration via `runtime parameters`.
+
+## Syntax
+
+Runtime parameters are indicated by the `requires` keyword, followed by the identifier of the parameter. Example
+
+```javascript
+block CarsLoader oftype SQLiteLoader {
+  table: "Cars";
+  file: requires CARS_SQLITE_FILE;
+}
+```
+
+## CLI
+
+The interpreter CLI has to define all existing runtime parameters for execution. 
+Use the CLI flag `-e` to to define them as key-value pairs of identifier and value.
+
+```console
+jv -e <param>=<value> -e <param>=<value> ... <file>
+```
+
+
+## JV Hub
```

Comment:
I think we should not mention the hub in the Jayvee documentation. Such information should rather be part of the hub documentation.

Reply:
Agreed.